docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET"
+description: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+ms.date: 04/17/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use cross validation to train a machine learning model in ML.NET
+---
+
+# How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+
+Learn how to use cross validation to build more robust machine learning models in ML.NET. Although this sample uses a regression model, it is applicable to other algorithms in ML.NET.
+
+## The data and data model
+
+Given data from a file that has the following format:
+
+```text
+Size (Sq. ft.), HistoricalPrice1 ($), HistoricalPrice2 ($), HistoricalPrice3 ($), Current Price ($)
+620.00, 148330.32, 140913.81, 136686.39, 146105.37
+550.00, 557033.46, 529181.78, 513306.33, 548677.95
+1127.00, 479320.99, 455354.94, 441694.30, 472131.18
+1120.00, 47504.98, 45129.73, 43775.84, 46792.41
+```
+
+The data can be modeled by a class like `HousingData`:
+
+```csharp
+public class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+ 
+    [LoadColumn(1, 3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+
+    [LoadColumn(4)]
+    [ColumnName("Label")]
+    public float CurrentPrice { get; set; }
+}
+```
+
+Load the data in into an [`IDataView`](xref:Microsoft.ML.IDataView).
+
+## Prepare the data
+
+Pre-process the data before using it to build the machine learning model. In this sample, the `Size` and `HistoricalPrices` columns are combined into a single feature vector,  which is output to a new column called `Features` using the [`Concatenate`](xref:Microsoft.ML.TransformExtensionsCatalog.Concatenate*) method. Then, [`NormalizeMinMax`](xref:Microsoft.ML.NormalizationCatalog.NormalizeMinMax*) is applied to the `Features` column to get `Size` and `HistoricalPrices` in the same range between 0-1.
```

Comment:
>) is applied to the `Features` column  [](start = 400, length = 38)

do you think it is a good place to stress that:
1-  concatenating data of the same time, before applying normalization speeds up training, because it does one pass over the data, taking care of normalizing both columns.

Reply:
#resolved